tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +55,4 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == False
```

Comment:
Great start! It would be good to also have an assert to make sure the number of items in the inventory hasn't changed and that nothing was accidentally removed.